aws/ecs_cluster.go

```diff
@@ -11,14 +11,54 @@ import (
 	"github.com/gruntwork-io/gruntwork-cli/errors"
 )
 
+// Used in this context to determine if the ECS Cluster is ready to be used & tagged
+// For more details on other valid status values: https://docs.aws.amazon.com/sdk-for-go/api/service/ecs/#Cluster
+const activeEcsClusterStatus string = "ACTIVE"
+
+// Filter all active ecs clusters
```

Comment:
NIT: Expand the comment a bit to explain why we only use ACTIVE clusters, as it took us a bit of back and forth to figure that out. 

BTW, what did you find on clusters in FAILED status?

Reply:
just realised I did not post my answer - started looking at it, and then jumped on a call. Will add shortly what I've found.

Reply:
Oops, I could swear there was a better comment when the actual filtering is done. I think I accidentally got rid of it while refactoring. I'm bringing it back & would leave it closer to the logic, rather than the const declaration, as it makes it a bit clearer in my view.